locale/en/docs/guides/domain-postmortem.md

```diff
@@ -360,8 +360,8 @@ const server = net.createServer((c) => {
   // Mock class that does some useless async data transformation
   // for demonstration purposes.
   const ds = new DataStream(dataTransformed);
-  c.on('data', (chunk) => ds.data(chunk));
-}).listen(8080, () => console.log(`listening on 8080`));
+  c.on('data', (chunk) => { ds.data(chunk); });
```

Comment:
It seems to me that omission of curly braces infers implicit return. Let me know if I should revert.